lxd/instance/drivers/driver_qemu.go

```diff
@@ -2331,7 +2331,7 @@ func (d *qemu) deviceDetachBlockDevice(deviceName string) error {
 
 	escapedDeviceName := filesystem.PathNameEncode(deviceName)
 	deviceID := fmt.Sprintf("%s%s", qemuDeviceIDPrefix, escapedDeviceName)
-	blockDevName := d.blockNodeName(escapedDeviceName)
+	blockDevName := d.generateQemuDeviceName(escapedDeviceName)
```

Comment:
Commit message "lxd/instance/drivers: Rename blockNodeName to generateQemuIdentifier" does not align with new function name "d.generateQemuDeviceName"

Reply:
Why is the function name renamed felt to be needed btw?

Reply:
The intent was mostly because it is not being used only for block nodes anymore, but also qemu disk device tags. Also I thought using just `generateDeviceName` was too broad because the same device has a different name on QEMU and on lxd.

Reply:
Thanks makes sense, please include this sort of thing in commit messages as it helps the reviewer understand the reason behind the changes. Ta